app/assets/javascripts/discourse/app/controllers/users.js

```diff
@@ -19,6 +19,14 @@ export default Controller.extend({
 
   showTimeRead: equal("period", "all"),
 
+  @discourseComputed("group")
+  selectedGroupId(group) {
+    let selectedGroup = this.get("availableGroups").find(
```

Comment:
also what does get accomplishes here ?

Reply:
Nothing special, ill remove it and go with the normal variable call